src/app/checkout/payment/templates/checkout.payment.html

```diff
@@ -1,6 +1,10 @@
 <div ng-form="OC_Checkout_Payment" novalidate>
     <div class="row">
         <div class="col-md-8">
+            <div class="form-group">
+                <label for="POIDInput"><h4>PO ID</h4></label>
+                <input id="POIDInput" placeholder="(Optional)" ng-model="base.currentOrder.xp.POID" type="text" class="form-control">
+            </div>
```

Comment:
How is this PO# number getting saved? Shouldn't there be an ng-blur or ng-change attached to a function to patch the order with that xp value?